src/framework/audio/internal/audiobuffer.cpp

```diff
@@ -19,13 +19,13 @@
 #include "audiobuffer.h"
 #include <cstring>
 #include "log.h"
+#include "synthtypes.h"
 
 using namespace mu::audio;
 
-AudioBuffer::AudioBuffer(unsigned int streamsPerSample, unsigned int size)
-    : m_streamsPerSample(streamsPerSample)
+AudioBuffer::AudioBuffer(unsigned int size)
 {
-    m_data.resize(size * m_streamsPerSample, 0.f);
+    m_data.resize(size * synth::AUDIO_CHANNELS, 0.f);
```

Comment:
This is not correct, or need to keep `m_streamsPerSample` (or rename to channels, streamsCount), or `AUDIO_CHANNELS` should be in `audio` namespace (not synth) and we deciding, what this our global limitation

Reply:
Yeah, I wasn't sure about it
Let's discuss it directly tomorrow, need some help with clarification